### ISSUE TITLE
Change brightness together with unchanged RGB value is not working

### ISSUE THREAD

**Issue by sstroot** [Foddy/node-red-contrib-huemagic#108]
**Describe the bug**
When I send a message with on, brightness and RGB and only the brightness is changing, the brightness is not sent to the hue lamp. When the brightness and RGB values changes, everything is working well.

**Flow to Reproduce**
```
[{"id":"290bd37d.cad59c","type":"inject","z":"3da7efa8.dfed4","name":"br100","topic":"","payload":"{\"on\":true,\"rgb\":[153,111,63],\"brightness\":\"100\"}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":820.2143402099609,"y":853.0000610351562,"wires":[["48cf52c6.eacd4c"]]},{"id":"e156b596.005388","type":"inject","z":"3da7efa8.dfed4","name":"br50","topic":"","payload":"{\"on\":true,\"rgb\":[153,111,63],\"brightness\":\"50\"}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":790.2143402099609,"y":912.0000610351562,"wires":[["48cf52c6.eacd4c"]]},{"id":"48cf52c6.eacd4c","type":"hue-light","z":"3da7efa8.dfed4","name":"Küche","bridge":"870de88f.3aee98","lightid":"22","colornamer":true,"x":908.214391708374,"y":617.5715255737305,"wires":[[]]},{"id":"870de88f.3aee98","type":"hue-bridge","z":"","name":"Philips hue","bridge":"172.16.0.140","key":"xyz","interval":"3000"}]
```

**Expected behavior**
Brightness is adjusted at the hue lamp.

**Please complete the following information:**
 - NodeRED version: 1.0.2
 - NodeJS version: 10.15.1
 - Device running NodeRED: Windows

https://app.leanboard.io/board/f91ab067-05b2-4e59-a4ef-4bfa7c5b6ed5

**Comment by Foddy** [comment]
If I understand you correctly, is a parameter ignored? In your case, does it ignore the brightness setting?

**Comment by sstroot** [comment]
@Foddy 
> If I understand you correctly, is a parameter ignored?

Yes, it is ignored. The lamps brightness is not changing 
> In your case, does it ignore the brightness setting?

Yes, because no other values are changing, the lamp does not change anything.

It seems like the brightness is only sending to the lamp, when the RGB value is changing too. Remember this is only when on, rgb and brightness is in the same msg.

**Comment by sstroot** [comment]
@Foddy I'm sorry, the problem was the IKEA TRADFRI blub. This is not a bug of this library.

**Comment by shopro** [comment]
I know this got closed, but I faced the same issue and possible found a working work around.
I can confirm there seems to be an issue regarding IKEA TRÅRFRI, at least the 
[TRÅDFRI LED bulb E12 600 lumen](https://www.ikea.com/us/en/p/tradfri-led-bulb-e12-600-lumen-wireless-dimmable-color-and-white-spectrum-opal-30411566/) is affected. Same issue regardless whether using the Hue Lights node or the Hue Groups node.

I am using the following work around to get the random colors and brightness working.
First we need to send just the desired brightness as the first payload, in example: `{"brightness":100}`

After that there needs to be a delay, I'm using the random node with 1-5s delay. I couldn't get it to work reliably without at least 1 second delay for some reason.

Then we need to send the second payload, in example: `{"color":"random","transitionTime":60}`

This has at least so far worked fine and does first set the brightness to desired level followed by smooth color change transition. It does not work for me when skipping the first brightness payload and instead adding brightness to the second payload.
